cfpb/consumerfinance.gov
# Wagtail: Add additional filter checks and add help text to Wagtail UI

## Changes

- Move filter by keyword checkbox down so it's nearer to the other filter checkboxes.
- Add help text and more readable labels to Wagtail UI for the individual filters.
- Add additional checks for `form.` values for topics and products.


## How to test this PR

1. Pull branch, run migrations, check a page like https://www.consumerfinance.gov/enforcement/actions/ in Wagtail and observe the help text for the filters.


## Screenshots

<img width="719" alt="Screen Shot 2021-07-06 at 5 48 08 PM" src="https://user-images.githubusercontent.com/704760/124670879-9e6d2580-de82-11eb-892e-2f310e1fd3f2.png">
